pkg/store/consul/rcstore/consul_store.go

```diff
@@ -663,6 +663,14 @@ func (s *ConsulStore) UpdateManifest(id fields.ID, man manifest.Manifest) error
 	return s.retryMutate(id, manifestUpdater)
 }
 
+func (s *ConsulStore) UpdateStrategy(id fields.ID, strategy fields.Strategy) error {
+	strategyUpdater := func(rc fields.RC) (fields.RC, error) {
```

Comment:
maybe here you should check that you got a real strategy

Reply:
although I didn't do that for pod clusters either so maybe i'm a hypocrite :P

Reply:
👍

Reply:
didn't you? https://github.com/square/p2/pull/954/files#diff-21e734f407a99256a47190a2322142b1R106

Reply:
ah yeah i meant i didn't do it in a store function, but I also didn't write a store function I just used `MutatePC()`.